### DISCUSSION TITLE
Using other p5 community libraries w/ p5-wrapper

### DISCUSSION THREAD

**Discussion by munusshih** [P5-wrapper/react#303] [Q&A]
Has anyone have experience using other p5 community libraries (such as p5.pattern or p5.brush) along with p5-wrapper?
And how did you do it?

I did read the example in [README](https://github.com/P5-wrapper/react), but found it hard to follow.
I was trying to use [p5.pattern](https://github.com/SYM380/p5.pattern) in my code by doing the following: 
```
(window as any).p5 = p5;
await import('./p5.pattern.js');
```

but when I try to use `p5.patternColors`, it still throws me error `TypeError: p5.patternAngle is not a function`.

**Comment by jamesrweb** [comment]
Hey @munusshih, thanks for reaching out.

How did you install p5 pattern?
Did you get errors in the console?
Did you get build errors?
Did you try any debugging steps before beginning the discussion? (useful to not duplicate pre-checked areas)
Can you share your repository?

**Comment by jamesrweb** [comment]
Closing due to inactivity.
